frontend/src/tests/import-export.test.js

```diff
@@ -107,6 +201,146 @@ describe("Import/export library functionality", () => {
         ).not.toEqual(targetData);
     });
 
-    it.todo("Export data to a JSON/CSV/XLSX");
-    it.todo("Import data from a JSON/CSV/XLSX");
+    it("Export data to a JSON/CSV/XLSX", () => {
+        // export instructor data a CSV File object
+        const fileCSV = getInstructorsDataFile("csv");
+        expect(fileCSV.name).toEqual(
+            `instructors_export_${new Date().toLocaleDateString("en-CA", {
+                year: "numeric",
+                month: "numeric",
+                day: "numeric",
+            })}.csv`
+        );
+
+        // export instructor data to a JSON File object
+        const fileJSON = getInstructorsDataFile("json");
+        expect(fileJSON.name).toEqual(
+            `instructors_export_${new Date().toLocaleDateString("en-CA", {
+                year: "numeric",
+                month: "numeric",
+                day: "numeric",
+            })}.json`
+        );
+
+        // export instructor data to a XLSX File object
+        const fileXLSX = getInstructorsDataFile("spreadsheet");
+        expect(fileXLSX.name).toEqual(
+            `instructors_export_${new Date().toLocaleDateString("en-CA", {
+                year: "numeric",
+                month: "numeric",
+                day: "numeric",
+            })}.xlsx`
+        );
+
+        // invalid file type, should throw error
+        expect(() => getInstructorsDataFile("nonsense")).toThrow(Error);
+
+        const resultCorrect = instructorData.map((instructor) =>
+            prepareMinimal.instructor(instructor)
+        );
+
+        // import instructor data from a JSON File object
+        const dataJSON = JSON.parse(fileJSON.buffer.toString()).instructors;
+        const resultJSON = normalizeImport(
+            {
+                fileType: "json",
+                data: dataJSON,
+            },
+            instructorSchema
+        );
+        expect(resultJSON).toEqual(resultCorrect);
+
+        // import instructor data from a XLSX File object
+        const workbook = XLSX.read(fileXLSX.buffer[0], { type: "array" });
+        const sheet = workbook.Sheets[workbook.SheetNames[0]];
+        let dataXLSX = XLSX.utils.sheet_to_json(sheet, { header: 1 });
+        // transform to array of objects
+        const keys = dataXLSX.shift();
+        dataXLSX = dataXLSX.map(function (row) {
+            return keys.reduce(function (obj, key, i) {
+                obj[key] = row[i];
+                return obj;
+            }, {});
+        });
+        const resultXLSX = normalizeImport(
+            {
+                fileType: "spreadsheet",
+                data: dataXLSX,
+            },
+            instructorSchema
+        );
+        expect(resultXLSX).toEqual(resultCorrect);
```

Comment:
I think we can trust the `xslx` to be working correctly as long as the CSV is working correctly.